skills/apple/swiftui-whats-new-27/references/content-builder.md

```
@@ -0,0 +1,404 @@
+# ContentBuilder Unification
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)